R-packages/evalcast/R/evaluate_predictions.R

```diff
@@ -1,103 +1,51 @@
-#' Create a data frame of score cards
+#' Create a score card data frame
 #'
-#' Performs backtesting, through the following steps:
+#' Performs testing, through the following steps:
```

Comment:
Maybe this should say "evaluates the quality of a forecaster" rather than "performs testing".

Reply:
Done